libevm/JitVM.cpp

```diff
@@ -14,11 +13,135 @@ namespace dev
 {
 namespace eth
 {
+namespace
+{
+
+static_assert(sizeof(Address) == sizeof(evm_hash160),
```

Comment:
I kind of remember that static_assert is not available on all our platforms. That might have changed if we move to a newer msvc version.